docs/robot/CumulusCI_Library.html

```diff
@@ -5,8 +5,8 @@
 <meta http-equiv="Pragma" content="no-cache">
```

Comment:
i have to admit i'm not fully sure why these files need to be committed, is there a way for readthedocs to generate them with sphinx? ... again, not in this PR ;)

Reply:
Ideally we should have a process that puts them in the gh-pages branch so they can just be linked to from the docs.  I just realized we could host static html there recently.

Reply:
Sounds good, can we handle that separately?